src/app/modules/filters/filters-content/filters-content.component.html

```diff
@@ -0,0 +1,21 @@
+<div>
+    <div class="filter">
+        <ccf-checkbox
+            label="Technologies"
+            [options]="[ 'IMS', 'MxIF', 'PAS' ]"
+            (selectionChange)="updateTechnologies($event)"
```

Comment:
You don't need to have a whole bunch of one off functions like this. Do something like:
`<ccf-checkbox (selectionChange)="updateFilter('technology', $event)"`

Reply:
that's the way I was going to do it and Daniel said to do it this way